setting asax scratch directory
 
============================================================
=====         Summary of your script job               =====
============================================================
  The script file is: run_analysis.sh
  The time limit is 4:00:00 HH:MM:SS.
  The target directory is: /home/uahoxa001/WLASL/start_kit
  The working directory is:  /scratch-local/uahoxa001.nlpinput4.99222.asax-pbs1
  The memory limit is: 4gb
  The job will start running after: 202408221459.44
  Job Name: nlpinput4
  Queue: -q express
  Constraints: 
  Using  2  cores on master node  asax013.asc.edu
  Node list:  asax013.asc.edu asax013.asc.edu
  Cores:  2
  Command typed:
/scripts/run_script run_analysis.sh     
  Queue submit command:
qsub -q express -j oe -N nlpinput4 -a 202408221459.44 -r n -M oaa0008@uah.edu -l walltime=4:00:00 -l select=ncpus=2:mpiprocs=2:mem=4000mb 

Lmod is automatically replacing "anaconda/3-2024.02" with "pytorch/2019".

[nltk_data] Error loading punkt: <urlopen error [SSL:
[nltk_data]     CERTIFICATE_VERIFY_FAILED] certificate verify failed
[nltk_data]     (_ssl.c:852)>
[nltk_data] Error loading averaged_perceptron_tagger: <urlopen error
[nltk_data]     [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify
[nltk_data]     failed (_ssl.c:852)>
Traceback (most recent call last):
  File "input_nlp.py", line 57, in <module>
    sia = SentimentIntensityAnalyzer()
  File "/apps/x86-64/apps/pytorch_2019/lib/python3.6/site-packages/nltk/sentiment/vader.py", line 200, in __init__
    self.lexicon_file = nltk.data.load(lexicon_file)
  File "/apps/x86-64/apps/pytorch_2019/lib/python3.6/site-packages/nltk/data.py", line 834, in load
    opened_resource = _open(resource_url)
  File "/apps/x86-64/apps/pytorch_2019/lib/python3.6/site-packages/nltk/data.py", line 952, in _open
    return find(path_, path + ['']).open()
  File "/apps/x86-64/apps/pytorch_2019/lib/python3.6/site-packages/nltk/data.py", line 673, in find
    raise LookupError(resource_not_found)
LookupError: 
**********************************************************************
  Resource [93mvader_lexicon[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('vader_lexicon')
  [0m
  Searched in:
    - '/home/uahoxa001/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
    - '/apps/x86-64/apps/pytorch_2019/nltk_data'
    - '/apps/x86-64/apps/pytorch_2019/lib/nltk_data'
    - '/home/uahoxa001/nltk_data'
    - ''
**********************************************************************

